lib/yardstick/v2_client/grant.rb

```diff
@@ -0,0 +1,40 @@
+require 'yardstick/v2_client/remote_model'
+require 'yardstick/cacheable'
+
+module Yardstick
+  module V2Client
+    class Grant
+      include RemoteModel
+      include Cacheable
+
+      cached_accessor :id
+
+      attr_accessor :metadata,
+                    :start_at,
+                    :expire_at,
+                    :allow_sittings_start_at,
+                    :allow_sittings_end_at,
+                    :time_zone_name,
+                    :exam_form_id,
+                    :send_email,
+                    :exam_form_group,
+                    :num_uses
+
+      def self.find(token, user_id, id)
+        resource_uri "/v2/users/#{user_id}/grants"
```

Comment:
NO!

This is very much not threadsafe...

`resource_uri` is not meant to be called in a method... and it's never meant to change.. It's a class level variable too... so any subsequent calls that DON'T call resource_uri is just going to always get grants for whatever use was last called...